fluentd/fluent.conf

```diff
@@ -112,8 +112,10 @@
   tag system.var.log
   format multiline
   format_firstline /^[A-Z][a-z]{2} [ |0-3]+[0-9] [0-2][0-9]:[0-5][0-9]:[0-5][0-9] /
-  format1 /^(?<time>[^ ]* [^ ]* [^ ]*) (?<host>[^ ]*) (?<ident>[a-zA-Z0-9_\/\.\-]*)(?:\[(?<pid>[0-9]+)\])?[^\:]*\: *(?<message>.*)$/
+  format1 /^(?<time>[^ ]*[ ]*[^ ]*[ ]*[^ ]*) (?<host>[^ ]*) (?<ident>[a-zA-Z0-9_\/\.\-]*)(?:\[(?<pid>[0-9]+)\])?[^\:]*\: *(?<message>.*)$/
```

Comment:
I definitely want + instead of * for the white space... or were you thinking more replacing every instance of * with +?
I also should update line 114 to remove +

I can also update to use \s and \S.  I wasn't sure if it would make it easier for most to read or not